### PR TITLE
update contributors

### PR DESCRIPTION
**Pull request by weissi**

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -78,6 +85,7 @@ needs to be listed here.
 - Tim Condon <0xtimc@gmail.com>
 - Tim Kientzle <tkientzle@apple.com>
 - Tomer Doron <tomerd@apple.com>
+- Trevör Anne Denise <adtrevor@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @adtrevor if you would like me to add a proper email address here, please reply with the email address you'd like to have here.

File: CONTRIBUTORS.txt [hunk 2]
```diff
@@ -50,13 +54,16 @@ needs to be listed here.
...
 - Lev Walkin <vlm@lionet.info>
+- Liam Flynn <l.flynn@live.co.uk>
 - Ludovic Dewailly <me@ldewailly.com>
 - Markus Jais <markusjais@googlemail.com>
+- Markus Kieselmann <markus@kieselmann.eu>
 - Matt Eaton <agnosticdev@gmail.com>
 - Max Desiatov <max.desiatov@gmail.com>
 - Max Moiseev <moiseev@apple.com>
 - Moritz Lang <hi@slashmo.codes>
 - Nathan Harris <Mordil@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @Mordil  if you would like me to add a proper email address here, please reply with the preferred email address.

File: CONTRIBUTORS.txt [hunk 3]
```diff
@@ -14,24 +14,27 @@ needs to be listed here.
 - 0xflotus <0xflotus@gmail.com>
 - Adam Nemecek <adamnemecek@gmail.com>
 - Albert Aleksieiev <albert.aleksieiev@gmail.com>
-- Alex Blewitt <alex.blewitt@gmail.com>
+- Alex Blewitt <alblue@apple.com>
+- Alex Reilly <fabiobean2@gmail.com>
 - Andrew Bennett <635596+therealbnut@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @therealbnut  if you would like me to add a proper email address here, please reply with the preferred email address.

File: CONTRIBUTORS.txt [hunk 4]
```diff
@@ -50,13 +54,16 @@ needs to be listed here.
...
+- Markus Kieselmann <markus@kieselmann.eu>
 - Matt Eaton <agnosticdev@gmail.com>
 - Max Desiatov <max.desiatov@gmail.com>
 - Max Moiseev <moiseev@apple.com>
 - Moritz Lang <hi@slashmo.codes>
 - Nathan Harris <Mordil@users.noreply.github.com>
+- Nathan Harris <nathan@mordil.info>
 - Nethra Ravindran <nethra.ravindran@in.ibm.com>
 - Norio Nomura <norio.nomura@gmail.com>
 - Norman Maurer <norman_maurer@apple.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @portia-s  if you would like me to add a proper email address here, please reply with the preferred email address.

Comment (weissi) [inline CONTRIBUTORS.txt]: @arukshani  if you would like me to add a proper email address here, please reply with the preferred email address.

File: CONTRIBUTORS.txt [hunk 5]
```diff
@@ -78,6 +85,7 @@ needs to be listed here.
 - Tim Condon <0xtimc@gmail.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @SebastianThiebaud  if you would like me to add a proper email address here, please reply with the preferred email address.

Comment (weissi) [inline CONTRIBUTORS.txt]: @broadwaylamb  if you would like me to add a proper email address here, please reply with the preferred email address.

Comment (weissi) [inline CONTRIBUTORS.txt]: @ogres  if you would like me to add a proper email address here, please reply with the preferred email address.

Comment (weissi) [inline CONTRIBUTORS.txt]: @siemensikkema  if you would like me to add a proper email address here, please reply with the preferred email address.

Comment (ogres) [reply CONTRIBUTORS.txt]: Proper email would be _beruashvili@gmail.com_, Thanks!

Comment (weissi) [reply CONTRIBUTORS.txt]: thanks! done

File: CONTRIBUTORS.txt (hunk 2 above)
Comment (weissi) [reply CONTRIBUTORS.txt]: ignore me, you were already in there with a proper email, de-duped

File: CONTRIBUTORS.txt (hunk 5 above)
Comment (broadwaylamb) [reply CONTRIBUTORS.txt]: sure!
jaskiewiczs@icloud.com

Comment (weissi) [reply CONTRIBUTORS.txt]: thanks, done.

File: CONTRIBUTORS.txt (hunk 3 above)
Comment (therealbnut) [reply CONTRIBUTORS.txt]: Hi,
andrew@nio.bennnn.net
thanks :)

File: CONTRIBUTORS.txt (hunk 5 above)
Comment (siemensikkema) [reply CONTRIBUTORS.txt]: Ok, thanks! s.h.sikkema@gmail.com please :)

File: CONTRIBUTORS.txt (hunk 3 above)
Comment (weissi) [reply CONTRIBUTORS.txt]: thanks, done!

File: CONTRIBUTORS.txt (hunk 5 above)
Comment (weissi) [reply CONTRIBUTORS.txt]: thanks, done

**Pull request merged**